### DISCUSSION TITLE
TypeError: prompt_async() got an unexpected keyword argument 'prompt_text'

### DISCUSSION THREAD

**Discussion by dbbice** [acheong08/ChatGPT#1113] [General]
TypeError: prompt_async() got an unexpected keyword argument 'prompt_text'
新的V2版本是不是不需要登录账号和密码了，运行V2.py报错有人知道怎么解决吗

**Comment by acheong08** [comment]
Strange. No such line exists `prompt_text`

**Comment by acheong08** [comment]
V2 has been disabled due to OpenAI patch. They're watching and working quick

**Reply by acheong08** [reply]
But V1 works good now
